run_full_pipe.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# this is a gold standard reproducible pipeline for the analysis of Live-Cell timelapse imaging data
+
+
+############################################
+# Download and pre-process the data
+############################################
+echo "Starting the data download and pre-processing..."
+
+# Download the data
+# TODO: Add the link to the data once it is available
```

Comment:
Would this comment be better served as a GitHub issue instead of a TODO (which might go more easily overlooked)?

Reply:
Issue referenced!